ui/pages/confirmations/components/gas-timing/gas-timing.component.js

```diff
@@ -65,6 +65,7 @@ export default function GasTiming({
   const previousIsUnknownLow = usePrevious(isUnknownLow);
```

Comment:
Looks like we're not reading `gasFeeEstimates` and `gasEstimateType` from the `gasFeeEstimatesByChainId` state yet on this component. Is that intentional?

Reply:
i believe values derived from gasFeeEstimatesByChainId are used here